pages/api/inserts/schedule.ts

```diff
@@ -33,16 +33,12 @@ export default async function handler(
   const name = req.query.name;
   if (typeof name === "string" && req.body != null) {
     // Inserts Data into the Schedule page
-    const sqlRes = await con.run(
+    await con.run(
       `INSERT INTO schedule (userID, name, scheduleData) 
         VALUES (?, ?, ?) 
         ON CONFLICT(userID, name) DO UPDATE SET scheduleData = excluded.scheduleData, timestamp = datetime('now', 'localtime')`,
       [user, name, req.body]
     );
-    if (sqlRes.changes !== 1) {
-      res.status(500).json({ error: "Internal Database Error" });
-      return;
-    }
```

Comment:
Why are we removing an error message? I assume there's a reason...

Reply:
I can't test this, since it never happens. The only way for the changes to be anything other than one is for a DB error to occur, but a DB error would actually trigger an exception, not return .changes !== 1.